core/src/test/java/org/hibernate/ogm/backendtck/associations/collection/types/MapTest.java

```diff
@@ -34,6 +37,23 @@
 public class MapTest extends OgmTestCase {
 
 	@Test
+	@TestForIssue(jiraKey = "OGM-969")
+	// ORM does not support mapping different associations of the same entity using the same table.
+	// Each association must have a different table. For this reason we decided to update the tests and rename the tables
+	// I'll leave this here just in case someone decides to change the test.
+	public void testDefaultAssociationKeyMetadataEquals() throws Exception {
```

Comment:
Can we move that out of the TCK? I don't think it needs to run for every dialect.